library/dnet.c

```diff
@@ -1799,9 +1799,22 @@ static int n2_process_cmd_with_backend_raw(struct dnet_net_state *st,
 			if (err != -ENOTSUP && err != -ENOENT)
 				break;
 		}
+		cmd->flags &= ~DNET_FLAGS_NEED_ACK;
```

Comment:
Here it's already done: https://github.com/interiorem/elliptics/blob/4fc279c5a90efc4e2b2e68bd4c2d44448f28ea1f/library/old_protocol/repliers.cpp#L19

In old mechanic `DNET_FLAGS_NEED_ACK` was unset manually because it is incompatible with `DNET_FLAGS_REPLY`, but in new mechanic (after PR 272) protocol itself setups `dnet_cmd` for reply.

Reply:
Ok, I'll revert it. Thank you.

And I see, that now `replier_base::replier_base` change `cmd_.flags` for all commands.
In old variant of code we did it only for READ/WRITE/LOOKUP.
We will need to fix this line when we start to refactor other commands.

Reply:
Ok, we'll research it. Very mystic behavior in old variant =)